root/types/entity.js

```diff
@@ -7,6 +7,20 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
+declare type AnnotatableEntityT =
+  | AreaT
+  | ArtistT
+  | EventT
+  | GenreT
+  | InstrumentT
+  | LabelT
+  | PlaceT
+  | RecordingT
+  | ReleaseT
+  | ReleaseGroupT
+  | SeriesT
+  | WorkT;
+
```

Comment:
@reosarevok wrote in https://github.com/metabrainz/musicbrainz-server/commit/f4065f0bf562734bc4111d60301db8f66fd6dd50#r103558097:

> There's already `AnnotatedEntityT` in `annotation.js`